Add start-time and end-time attributes to test-cases and test-suites

Nikheln opened the pull request:
This PR adds the start-time and end-time attributes specified in the NUnit XML schema to test-case and test-suite elements. I did not find a reasonable way to accurately get test suite start and end times from the test framework, so those properties are aggregated based on their contained cases' or suites' start and end times.

codito commented:
@Nikheln thank you for the patch. Changes look good to me! Would you mind adding a few tests please?

Nikheln commented:
Good to hear! I've updated the PR with a test to validate the start and end times of one test (I also updated the corresponding test case with a longer delay to cause a difference to the start and end times), and also added timestamp verification to the test suite attribute test.

codito commented:
Latest package should be available from our myget feed: https://www.myget.org/feed/spekt/package/nuget/NunitXml.TestLogger

I will cut a nuget release in a day or two.
